### DISCUSSION TITLE
Can I receive responses from the Gemini Ultra or Pro version?

### DISCUSSION THREAD

**Discussion by hyeondori** [dsdanielpark/Gemini-API#18] [Q&A] [answered]
Currently, the standard version of Gemini is being serviced on the web. Is there a possibility to use Gemini Ultra (Gemini Advanced)? I would like to utilize Gemini Ultra (Gemini Advanced) through reverse engineering.

**Comment by dsdanielpark** [comment, marked as answer]
You can only get responses from the services currently available on the web. If you wish to use Gemini Ultra (Gemini Advanced), you’ll need to set up an application in Google authentication and opt for a paid plan. This package only returns information about the text provided by the web service. Thank you for the good response!

**Reply by dsdanielpark** [reply]
However, if it becomes possible in the future, I will update it as soon as I can.

**Comment by bluemods** [comment]
You can opt to use the Advanced or non-advanced model by modifying a field in the StreamGenerate request.

1 = Standard
2 = Advanced

Request payload samples attached, it is easy to spot the difference.
[samples.txt](https://github.com/dsdanielpark/Gemini-API/files/14750098/samples.txt)

Note that the API will return an error when attempting to use Advanced on an account that doesn't own it, so clients need to handle this properly.

**Reply by dsdanielpark** [reply]
I will check and add experimental featurs ASAP! Thank you.
